<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monsters</class>
 <widget class="QWidget" name="Monsters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="6">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_6"/>
       </widget>
      </item>
      <item row="7" column="2" colspan="3">
       <widget class="QLineEdit" name="custom_name"/>
      </item>
      <item row="0" column="2" colspan="3">
       <widget class="QComboBox" name="search">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="5">
       <widget class="QFrame" name="image_3">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="2" column="5">
          <widget class="QLineEdit" name="defensive_magic"/>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="defensive_stab"/>
         </item>
         <item row="1" column="6">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../images/monster/Ranged_icon.webp</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="defensive_slash"/>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../images/monster/White_scimitar.webp</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../images/monster/White_warhammer.webp</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../images/monster/Magic_icon.webp</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="5">
          <widget class="QFrame" name="frame_6">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <layout class="QGridLayout" name="gridLayout_9"/>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../images/monster/Defence_icon.webp</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLineEdit" name="defensive_crush"/>
         </item>
         <item row="2" column="6">
          <widget class="QLineEdit" name="defensive_ranged"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../images/monster/White_dagger.webp</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1" rowspan="3">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string> Defensive Stats </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1" colspan="5">
       <widget class="QFrame" name="image">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="3" column="4">
          <widget class="QLineEdit" name="strength"/>
         </item>
         <item row="2" column="6">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../images/monster/Magic_icon.webp</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../images/monster/Hitpoints_icon.webp</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="7">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../images/monster/Ranged_icon.webp</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1" rowspan="3">
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="6">
          <widget class="QLineEdit" name="magic"/>
         </item>
         <item row="3" column="3">
          <widget class="QLineEdit" name="attack"/>
         </item>
         <item row="2" column="4">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../images/monster/Strength_icon.webp</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="5">
          <widget class="QLineEdit" name="defence"/>
         </item>
         <item row="2" column="0">
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>../images/monster/Combat_icon.webp</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../images/monster/Defence_icon.webp</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="health"/>
         </item>
         <item row="3" column="7">
          <widget class="QLineEdit" name="ranged"/>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../images/monster/Attack_icon.webp</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="combat_level"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>   Combat Stats  </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="1" colspan="5">
       <widget class="QFrame" name="image_2">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="2">
          <widget class="QLineEdit" name="aggressive_attack"/>
         </item>
         <item row="2" column="5">
          <widget class="QLineEdit" name="aggressive_magic_damage"/>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../images/monster/Strength_icon.webp</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1" rowspan="3">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="6">
          <widget class="QFrame" name="frame_5">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <layout class="QGridLayout" name="gridLayout_8"/>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../images/monster/Attack_icon.webp</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="6">
          <widget class="QLineEdit" name="aggressive_ranged"/>
         </item>
         <item row="1" column="6">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../images/monster/Ranged_icon.webp</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../images/monster/Magic_icon.webp</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="aggressive_strength"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../images/monster/Attack_icon.webp</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../images/monster/Magic_Damage_icon.webp</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../images/monster/Ranged_Strength_icon.webp</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="7">
          <widget class="QLineEdit" name="aggressive_ranged_strength"/>
         </item>
         <item row="2" column="4">
          <widget class="QLineEdit" name="aggressive_magic"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Aggressive Stats</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="nmz_only">
        <property name="text">
         <string>NMZ only</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>Custom Name</string>
        </property>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="QPushButton" name="add">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="wiki_link">
        <property name="text">
         <string>See on Wiki</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>xp/damage</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="xp_per_hit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
